### PR TITLE
Fix consumers sorting

### PR DESCRIPTION
**Pull request by Haarolean**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Review by workshur — APPROVED** [review]

**Review by iliax — APPROVED** [review]

**Comment by shubhwip** [comment]
As per the #2445 Consumer sorting is broken for `Members` Case.

> /api/clusters/local/consumer-groups/paged?page=1&perPage=25&search=&orderBy=**MEMBERS**&sortOrder=DESC
> returns payload in ASC order


Originally `getPaginationComparator(orderBy)` was returning `ascending` comparator by default except for `MEMBERS` case because in `MEMBERS` case we were returning `descending` comparator `Comparator.comparingInt(cg -> -cg.members().size());` 

So is it a good idea to return the `ascending comparator` for DESC case for all `orderBy` queries to fix only `MEMBERS` sorting or i am missing something :) ?

**Comment by Haarolean** [comment]
> As per the #2445 Consumer sorting is broken for `MEMBERS` Case.
> 
> > /api/clusters/local/consumer-groups/paged?page=1&perPage=25&search=&orderBy=**MEMBERS**&sortOrder=DESC
> > returns payload in ASC order
> 
> Originally `getPaginationComparator(orderBy)` was returning `ascending` comparator by default except for `MEMBERS` case because in `MEMBERS` case we were returning `descending` comparator `Comparator.comparingInt(cg -> -cg.members().size());`
> 
> So is it a good idea to return the `ascending comparator` for DESC case for all `orderBy` queries to fix only `MEMBERS` sorting or am i missing something :) ?

thanks for the comment!
I tried one for `STATE` sorting as well and it was reversed as well:
`$.consumerGroups.*.state`
`http://master.internal.kafka-ui.provectus.io/api/clusters/local/consumer-groups/paged?page=1&perPage=25&search=&orderBy=STATE&sortOrder=DESC`:
```
[
	"EMPTY",
	"EMPTY",
	"EMPTY",
	"EMPTY",
	"STABLE",
	"STABLE",
	"STABLE",
	"STABLE",
	"STABLE",
	"STABLE",
	"STABLE",
	"STABLE"
]
```
`http://master.internal.kafka-ui.provectus.io/api/clusters/local/consumer-groups/paged?page=1&perPage=25&search=&orderBy=STATE&sortOrder=ASC`:
```
[
	"STABLE",
	"STABLE",
	"STABLE",
	"STABLE",
	"STABLE",
	"STABLE",
	"STABLE",
	"EMPTY",
	"EMPTY",
	"EMPTY",
	"EMPTY",
	"EMPTY"
]
```

**Comment by shubhwip** [comment]
@Haarolean Thanks for sharing API responses. As per your API output it seems correct to me because value of Consumer Group State `EMPTY` is 3 and value of `STABLE` is 0. 
Thus for sortOrder `DESC`, `EMPTY` will come before `STABLE` and for sortOrder `ASC`, `EMPTY` will come after `STABLE` as per the original implementation.

**Comment by Haarolean** [comment]
@shubhwip thanks, my bad! PTAL

File: kafka-ui-api/src/test/java/com/provectus/kafka/ui/container/KafkaConnectContainer.java [hunk 1]
```diff
@@ -12,6 +12,7 @@ public class KafkaConnectContainer extends GenericContainer<KafkaConnectContaine
   public KafkaConnectContainer(String version) {
     super("confluentinc/cp-kafka-connect:" + version);
     addExposedPort(CONNECT_PORT);
+
```
Comment (shubhwip) [inline kafka-ui-api/src/test/java/com/provectus/kafka/ui/container/KafkaConnectContainer.java:15]: This empty line could be removed :).

**Review by shubhwip — CHANGES_REQUESTED** [review]
LGTM 👍  with few minor changes :) !!

File: kafka-ui-api/src/test/java/com/provectus/kafka/ui/container/KafkaConnectContainer.java (hunk 1 above)
Comment (Haarolean) [reply kafka-ui-api/src/test/java/com/provectus/kafka/ui/container/KafkaConnectContainer.java:15]: I like the spacing like separating logical blocks

File: kafka-ui-api/src/test/java/com/provectus/kafka/ui/container/KafkaConnectContainer.java [hunk 2]
```diff
@@ -37,6 +38,7 @@ public KafkaConnectContainer withKafka(Network network, String bootstrapServers)
     withEnv("CONNECT_INTERNAL_KEY_CONVERTER", "org.apache.kafka.connect.json.JsonConverter");
     withEnv("CONNECT_INTERNAL_VALUE_CONVERTER", "org.apache.kafka.connect.json.JsonConverter");
     withEnv("CONNECT_REST_ADVERTISED_HOST_NAME", "kafka-connect");
+    withEnv("CONNECT_REST_PORT", "8083");
```
[the remark this answers is not in the document]
Comment (Haarolean) [reply kafka-ui-api/src/test/java/com/provectus/kafka/ui/container/KafkaConnectContainer.java:41]: sure

**Review by shubhwip — APPROVED** [review]
LGTM

**Pull request merged**